### PR TITLE
Separate collision radiuses for head and body

### PR DESCRIPTION
**Pull request by kmatheussen**
Patch says it all. We don't have to be so afraid of the body of other snakes. They don't move. It's pretty safe to move close to a snake body as long as we don't touch it. It's the heads we should run away from.

### CODE REVIEW

**Comment by Evanito** [comment]
Tested and worked as expected. :+1:

**Comment by Seple** [comment]
Super!
Please add `var speedingMultiplier = (window.snake.sp > 10) ? 1.0 : 1.0;`
This solved my problem #94        (aggressively fighting for eating dead snake)

**Comment by ChadSki** [comment]
@Seple Why don't you submit your own pull request to solve your issue?

@kmatheussen The commit looks good! :+1: I'm looking at the Hound complaints and they seem a little aggressive... @ErmiyaEskandary just enabled them.

File: bot.user.js [hunk 1]
```diff
@@ -373,58 +373,100 @@ var bot = (function() {
...
+            bot.closestHeadPoint = null;
+
+            var closestHeadPointDistance = Number.MAX_VALUE;
             var collisionPoints = [];
-            for (var snake in window.snakes) {
-                if (window.snakes[snake].nk != window.snake.nk) {
-                    for (var pts in window.snakes[snake].pts) {
-                        if (!window.snakes[snake].pts[pts].dying) {
+
+            for (var snakenum in window.snakes) {
```
Comment (ChadSki) [inline bot.user.js]: I hadn't heard of this rule before, so I looked it up and found this: http://stackoverflow.com/a/4166584/1628916

On the other hand all you did was rename a variable...

@ErmiyaEskandary do you think the bot is being too aggressive?

**Comment by ermiyaeskandary** [comment]
@ChadSki I'm removing it ... ;)

**Comment by ermiyaeskandary** [comment]
Approved.

[Approved with PullApprove](https://pullapprove.com/ErmiyaEskandary/Slither.io-bot/pull-request/131/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ErmiyaEskandary/Slither.io-bot)

**Comment by ermiyaeskandary** [comment]
@kmatheussen Please rebase your branch and then we can merge.

**Comment by kmatheussen** [comment]
@ErmiyaEskandary Rebased now.

**Comment by kmatheussen** [comment]
Sorry, I force pushed to clean up log. Shouldn't have done that I guess.

**Comment by Seple** [comment]
@ChadSki Really? and what it is #94 like not reporting a problem?

**Pull request merged**